go-controller/pkg/ovn/gress_policy.go

```diff
@@ -250,8 +253,8 @@ func (gp *gressPolicy) delNamespaceAddressSet(name, portGroupName string) {
 // by the parent NetworkPolicy)
 func (gp *gressPolicy) localPodAddACL(portGroupName, portGroupUUID string) {
 	l3Match := gp.getL3MatchFromAddressSet()
-
 	var lportMatch, cidrMatch string
+
```

Comment:
If there's going to be a blank line here, the original version makes more sense (separating the code that sets `l3Match` from the code that sets `lportMatch` and `cidrMatch`)

Reply:
I removed the blank line. Also I have not changed anything with the order or method of applying ACLs. I have just made simple changes to incorporate dual stack.